renderer/components/Charts/DataRelated.tsx

```diff
@@ -20,7 +15,7 @@ export default function DataRelatedChart({
           width: "2%",
         }}
       >
-        <TopArrow value={dataPacked} color="#7BF05E" />
+        {typeof dataPacked === "number" && <TopArrow value={dataPacked} color="#7BF05E" />}
```

Comment:
Maybe it's simplier
{dataPacked != null && <TopArrow value={dataPacked} color="#7BF05E" />}
Both checks for null and undefined

Reply:
that is indeed a clever trick to catch both, I didn't know it was possible. But since many years, I've avoided the use of == and != in favor of === and !==, I believe eslint will also give an error if you use the the older form, but I guess there are some gems that we lose by always using === / !==. (e.g. > `"0" == 0` is  true)